extensions/net/ops_tls.rs

```diff
@@ -679,6 +682,10 @@ pub struct ConnectTlsArgs {
   hostname: String,
   port: u16,
   cert_file: Option<String>,
+  #[serde(
+    deserialize_with = "deno_core::deserialize_allow_insecure_certificates"
+  )]
```

Comment:
Again, I don't understand why a custom deserializer is needed here, please elaborate.